index.js

```diff
@@ -0,0 +1,42 @@
+const select = document.querySelector("select");
+const title = document.querySelector("#title")
+const year = document.querySelector("#release-year")
+const description = document.querySelector("#description")
+const input = document.querySelector("input")
+const ul = document.querySelector("ul")
+
+const createOptions = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+    res.data.forEach((films) => {
+      const option = document.createElement("option");
+      option.textContent = films.title;
+      option.value = films.id;
+      select.appendChild(option);
+    });
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+createOptions()
+
+select.addEventListener("change", async (e) => {
+    try {
+        const res = await axios.get(`https://ghibliapi.herokuapp.com/films/${e.target.value}`)
+        title.textContent = res.data.title
+        year.textContent = res.data.release_date
+        description.textContent = res.data.description
+        const form = document.querySelector("form")
+        form.addEventListener("submit", (e) => {
+            e.preventDefault()
+            const li = document.createElement("li")
+            li.innerHTML = `<b>${title.textContent}:</b> ${input.value}`
+            ul.appendChild(li)
+            input.value = ""
+        })
```

Comment:
This should be defined outside, what you are doing here is adding the event listener every time a user changes the title in the select dropdown. I see that you placed it here to get the title of the current film chosen, but we can also get the current film using the select / movieTitles variable to get the current value by using the .innerText property